packages/contracts/src/contracts/Unlock/IUnlockV11.sol

```diff
@@ -1,90 +1,94 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.5.17 <0.9.0;
 
-
 /**
  * @title The Unlock Interface
-**/
+ **/
 
-interface IUnlockV11
-{
+interface IUnlock {
```

Comment:
Should we alias this? so people can use either?

Reply:
I guess what's the way to include this from the package?

Reply:
yes better to keep the version here. We have an open issue for adding aliases to the package itself